todo_app/data/session_items.py

```diff
@@ -1,33 +1,39 @@
 from flask import session
 
 _DEFAULT_ITEMS = [
-    { 'id': 1, 'status': 'Not Started', 'title': 'List saved todo items' },
-    { 'id': 2, 'status': 'Not Started', 'title': 'Allow new items to be added' }
+    {'id': 1, 'status': 'Not Started', 'title': 'List saved todo items'},
+    {'id': 2, 'status': 'Not Started', 'title': 'Allow new items to be added'}
 ]
 
 
+def item_key(item):
+    return item.get('status')
+
+
 def get_items():
     """
     Fetches all saved items from the session.
 
     Returns:
         list: The list of saved items.
     """
-    return session.get('items', _DEFAULT_ITEMS.copy())
+    session_items = session.get('items', _DEFAULT_ITEMS.copy())
+
+    return sorted(session_items, key=item_key)
```

Comment:
If I was feeling very mean then this will list the Completed status before Not Started, but if we actually care we can add a `reverse=True` argument easily enough!